cluster/src/main/java/com/hartwig/pipeline/Arguments.java

```diff
@@ -1,13 +1,11 @@
 package com.hartwig.pipeline;
 
-import java.util.Optional;
```

Comment:
You're not using the `hmf_code_scheme` from the root of this repo. That is causing a lot of the lines changed. Can you go through all changes in the PR and make sure the correct formatting is applied?

Reply:
undid the formatting commits, it should mostly be gone now.